File: AGG2CPP/aggregation/student.cpp
#include"student.h"
#include<iostream>
#include<string>
using namespace std;
student::student(){

}
student::student(string id,string depart,int year){
    this->name=id;
    this->department=depart;
    this->batch_no=year;
}
student::student(const student&stud){
    name=stud.name;
    department=stud.department;
    batch_no=stud.batch_no;

}

File: AGGREGATION(JAVA)/student.java
public class student {
    String name;
    String bookname;

    public student() {
    }

    public student(String name, String Bookname) {
        this.name = name;
        this.bookname = Bookname;

    }

    public student(student diffrent) {
        name = diffrent.name;
        bookname = diffrent.bookname;

    }

}